search/search-domain/src/main/java/io/camunda/search/entities/DecisionDefinitionEntity.java

```diff
@@ -12,8 +12,8 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public record DecisionDefinitionEntity(
     String tenantId,
```

Comment:
💭  tenantId as first property makes it look like this is the PK. Is it maybe useful to change it that the PK is the first argument?